### PR TITLE
Add a simple saving texture function into save_obj

### PR DESCRIPTION
**Pull request by yihang99**

### CODE REVIEW

**Comment by BachiLi** [comment]
Can this be an option? i.e. there is a flag where the user can decide whether to save the materials. It would also be great if it can handle multiple materials.

**Comment by yihang99** [comment]
Fixed symmetric envmap bug

File: pyredner/save_mtl.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        f.write('newmtl mtl_1\n')
+
+        if m.diffuse_reflectance is not None:
+            texels = m.diffuse_reflectance.texels
+            if len(texels.size()) == 1:
+                f.write('Kd {} {} {}\n'.format(texels[0], texels[1], texels[2]))
+            else:
+                f.write('map_Kd Kd_texels.png\n')
+                pyredner.imwrite(texels.data.cpu(), path + '/Kd_texels.png')
+            #pyredner.imwrite(torch.randn(10, 10).abs(), path + '/tt.png')
```
Comment (BachiLi) [inline pyredner/save_mtl.py:25]: remove this line?

Comment (yihang99) [reply pyredner/save_mtl.py:25]: OHHHH yes, sorry that was for testing...

**Comment by BachiLi** [comment]
Thanks!

**Pull request merged**